commons-api/src/main/java/org/exoplatform/commons/api/notification/rest/WebNotificationRestService.java

```diff
@@ -96,15 +97,27 @@ public Response updateNotifications(@ApiParam(value = "operation", required = tr
         if (notificationId == null) {
           return Response.status(Response.Status.BAD_REQUEST).build();
         } else {
-          webNftService.markRead(notificationId);
+          NotificationInfo notification = webNftService.getNotificationInfo(notificationId);
+          if (currentUser.equals(notification.getTo())) {
+            webNftService.markRead(notificationId);
+          } else {
+            return Response.status(Response.Status.UNAUTHORIZED).build();
```

Comment:
could you please add a `log.debug` entry to notice this to Ops (This shouldn't happen at all but when it happens, this means whether it's a real tentative of attack or the product has a bug). Same as line 118.

( BTW, could you please format all added classes using [IDE Formatter](http://developer.exoplatform.org/#id-formatter-settings) )

Reply:
Ok for the first point I'll do it.

For the formater, my intellij is configured in accordance with the link you provide.
Can you explain what is the problem with the new class  ?

Reply:
Nothing serious, multiple new lines at the end of the file, no newline between methods ( see WebNotificationRestServiceTest.java )

Reply:
Does the formatter should remove new lines add the end of the file ? Does it add new lines between methods  ?
What about in your configuration ?

Reply:
In fact, the fomatter do that, but I need to ask him explicitly to format. 
Done !